js/react/utils/helpers.jsx

```diff
@@ -6,3 +6,11 @@ export const loadlGoogleClientLibraryScript = (src) => {
 	const body = document.getElementsByTagName('body')[0];
 	body.appendChild(tag);
 };
+
+export const chunkArray = (arr, chunkSize) => {
+	const chunkedArray = [];
+	for (let i = 0; i < arr.length; i += chunkSize) {
+		chunkedArray.push(arr.slice(i, i + chunkSize));
+	}
+	return chunkedArray;
+};
```

Comment:
```suggestion
export const chunkArray = (initialArray, chunkSize) => {
	const chunks = [];
	for (let index = 0; index < initialArray.length; index += chunkSize) {
		const chunk = initialArray.slice(index, index + chunkSize);
		chunks.push(chunk);
	}
	
	return chunks;
};
```

Minor readability improvements